cdec/installation-manager-core/src/test/java/com/codenvy/im/managers/TestConfigManager.java

```diff
@@ -109,7 +110,8 @@ public void testLoadDefaultSingleServerCdecConfig() throws Exception {
                                              "b=2\n" +
                                              "c=\n" +
                                              "d=3\\n4\n");
-        doReturn(properties).when(transport).download(endsWith("codenvy-single-server-properties/3.1.0"), any(Path.class));
+        doReturn(properties).when(transport).download(endsWith("codenvy-single-server-properties/3.1.0"), any(Path.class),
+                                                      MediaType.APPLICATION_OCTET_STREAM);
```

Comment:
Leads to _Mockito_ error `Unfinished stubbing detected`.
Fixup: `eq(MediaType.APPLICATION_OCTET_STREAM)`